src/MarginTrading.Backend.Services/TradingEngine.cs

```diff
@@ -442,6 +448,8 @@ private Task<Order> CloseActiveOrderByMatchingEngineAsync(Order order, OrderClos
                 return true;
             });
 
+            _volumeEquivalentService.EnrichClosingOrder(order);
```

Comment:
Do we need to enrich an order if it didn't match?

Reply:
Fixed